app/routes.py

```diff
@@ -1 +1,235 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Flask, Blueprint, jsonify, abort, make_response, request
+from app import db
+from app.models.goal import Goal
+from app.models.task import Task
+from datetime import date
+
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+#TASK ROUTES
+
+def validate_model(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        abort(make_response({"message": f"{model_id} is not a valid type ({type(model_id)}). Must be an integer)"}, 400))
+
+    model = cls.query.get(model_id)
+    
+    if not model:
+        abort(make_response({"message": f"{cls.__name__} {model_id} does not exist"}, 404))
+        
+    return model
+
+###
+
+@tasks_bp.route("", methods=['POST'])
+
+def create_task():
+
+    request_body = request.get_json()
+
+    if "title" not in request_body or "description" not in request_body or "is_complete" == False:
+        abort(make_response({"details": "Invalid data"}, 400))
+    
+    new_task = Task(title=request_body["title"],
+                description=request_body["description"])
+        
+    if "completed_at" in request_body:
+        new_task.completed_at=request_body["completed_at"]
+
+    db.session.add(new_task)
```

Comment:
Great job using `db.session.<method-name>`. Session gives us access to the follow:

- `db` is our app’s instance of SQLAlchemy.
- `session` represents our active database connection.
- By referencing `db.session` we can use SQLAlchemy’s methods to perform tasks like:
    - committing a change to a model
    - storing a new record of a model
    - deleting a record.
- By referencing `db.session.add()` you are able to use the SQLAlchemy method to store a new record of the task model